crates/modelardb_server/src/storage/mod.rs

```diff
@@ -597,9 +625,12 @@ pub(self) fn create_time_and_value_range_file_name(batch: &RecordBatch) -> Strin
         Uuid::new_v4()
     };
 
+    // TODO: Use part of the UUID or the entire Apache Arrow Flight URL to identify the edge.
```

Comment:
The port will be the same for each edge node, so for now maybe use something like the edge node's [`name`](https://docs.rs/sysinfo/latest/sysinfo/trait.SystemExt.html#tymethod.name) or [`host_name`](https://docs.rs/sysinfo/latest/sysinfo/trait.SystemExt.html#tymethod.host_name) from the [`sysinfo`](https://crates.io/crates/sysinfo) crate. The name should probably be cached instead of retrieving it from the system each time.

Reply:
The reason the port is used for now is that it is available in the platform, which is the part that needs to identify the edge during data transfers. The port will be unique for now since the only place we will have multiple edges is in the demo environment where each edge will run on a different port.

Using the name or any unique ID that is generated on the edge means that we need to have a way to share this ID with the platform. We could add an action that is called on startup to retrieve the ID but I think it would be better to wait until we have started the implementation of the new cloud architecture since I think the manager will be used for this step. I have added some text about this to the relevant work item.